### PR TITLE
WIP: ARKAI-37 Alfresco EE and distributed Alfresco

### PR DESCRIPTION
**Pull request by david-oc-miller**
Alfresco EE actually not much different from CE; only different artifact names and download sites.

Also, starting to support installing components on different hosts, since customers are asking for Alfresco on different hosts than ArkCase.

### CODE REVIEW

File: vagrant/provisioning/arkcase-ce-facts.yml [hunk 1]
```diff
@@ -342,11 +353,22 @@ default_database_password: "@rKc@3S!"
...
 alfresco_records_management_version: 3.0.a
 
+# Alfresco Enterprise Edition
+# for Alfresco EE you must have an Alfresco EE license file
+# (NOT included in this repository), and you must have downloaded
+# the installer artifacts from Alfresco's support portal and uploaded
+# them to an SFTP repository.  Also, run the `alfresco_ee` role instead of
+# `alfresco_ce`
+alfresco_ee_version: 6.1.1.1
+alfresco_install_governance_services: no
```
Comment (david-oc-miller) [inline vagrant/provisioning/arkcase-ce-facts.yml:369]: TODO: I am not honoring the alfresco_install_governance_services flag yet.

File: vagrant/provisioning/roles/activemq/templates/activemq.service [hunk 2]
```diff
@@ -14,6 +14,7 @@ RestartSec=10
 StandardOutput=syslog
 StandardError=syslog
 SyslogIdentifier=activemq
+Environment="JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk"
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/activemq/templates/activemq.service:17]: Alfresco 6 was compiled with Java 11, hence I need to install both java 11 and Java 8.

File: vagrant/provisioning/roles/alfresco-ee/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+
+# no real way to extract a task or block here since we need to know 
+# specifically whether the content services zip was just now 
+# downloaded.  However, since we are downloading a specific version,
+# we can just check whether the file already exists before downloading it
+
+- name: see if Alfresco EE content services already downloaded
+  become: yes
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/alfresco-ee/tasks/main.yml:8]: TODO: refactor this into a block, hopefully, so only one stat task and one sftp task

File: vagrant/provisioning/roles/alfresco/tasks/main.yml [hunk 4]
```diff
@@ -60,7 +58,7 @@
   replace:
     path: "{{ root_folder }}/app/alfresco/conf/catalina.properties"
     regexp: '^shared.loader=$'
-    replace: 'shared.loader=${catalina.base}/shared/classes'
+    replace: 'shared.loader=${catalina.base}/shared/classes,${catalina.base}/shared/lib/*.jar'
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/alfresco/tasks/main.yml:61]: Alfresco docs mention this shared/lib/\*.jar path, so I'm adding it here.

**Review by bojmil-armedia — APPROVED** [review]
Looks good.

**Pull request merged**